mailpoet/assets/js/src/automation/editor/components/modals/deactivate-modal.tsx

```diff
@@ -5,6 +5,39 @@ import { dispatch, useSelect } from '@wordpress/data';
 import { storeName } from '../../store';
 import { WorkflowStatus } from '../../../listing/workflow';
 
+export function DeactivateImmediatelyModal({ onClose }): JSX.Element {
```

Comment:
We should probably add a type for props for both of the modals, sth like:

```ts
type Props = {
  onClose: () => void;
}
```

Reply:
Addressed in 3ebb7b743